config/generator_macros.j2

```diff
@@ -16,6 +16,7 @@
 {% macro header(imports=[], ignore=[]) -%}
 import unittest
 
+import {{ exercise }}
```

Comment:
I would prefer not to add to this file as this affects *all* generated test suites. If you really need a module import over specific symbol imports, please include it explicitly in the template for this exercise.